src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -39,7 +41,10 @@ const Authenticate = (() => {
         $submit_table,
         $button_uns,
         $submit_status_uns,
-        $submit_table_uns;
+        $submit_table_uns,
```

Comment:
what is `uns`? maybe we can rename to a more descriptive variable name.

Reply:
@matin-binary it's been there since i started working on POI

Reply:
Alright then you may leave it as it is.